draft-ietf-rats-architecture.md

```diff
@@ -407,15 +407,15 @@ Execution environments that are designed specifically to be capable of claims co
 
 ## Layered Attestation Environments {#layered-attestation}
 
-By definition, the Attester role creates Evidence.
+By definition, the Attester role generates Evidence.
 An Attester may consist of one or
-more nested or staged environments, adding complexity to the architectural
-structure. 
-The unifying component is the root of trust and the nested, staged,
-or chained attestation Evidence produced. 
-The nested or chained structure
-includes Claims, collected by the Attester to aid in the assurance or
-believability of the attestation Evidence.
+more nested environments (layers).
+The root layers of an Attester are called roots of trust.
+In order to appraise Evidence produced by an Attester, the Verifier needs to trust the Attester's root of trust.
+Trust in the Attester's root of trust can be established either directly (e.g., the Verifier puts the root of trust's public key into its trust anchor store) or transitively via an Endorser (e.g., the Verifier puts the Endorser's public key into its trust anchor store).
+In layered attestation, a root of trust is the initial Attesting Environment.
+Claims can be collected from or about each layer, including roots of trust.
```

Comment:
"Claims can be collected from or about each layer, including roots of trust."

This is technically correct. On the glossary level we have the responsibility to make absolutely sure that the reader is aware that a root of trust cannot ever generate Evidence from Claims about itself.